[CHANGE ME] Re-generated to pick up changes from googleapis-discovery.

yoshi-automation opened the pull request:
This PR was generated using Autosynth. :rainbow:

Synth log will be available here:
https://source.cloud.google.com/results/invocations/bf198e5d-3166-4880-a527-31bd5c46ed48/targets

- [ ] To automatically regenerate this PR, check this box. (May take up to 24 hours.)

Source-Link: https://github.com/googleapis/googleapis-discovery/commit/c5bb8d324832f638d86bc1348fb45fb3d9c1fb30
Source-Link: https://github.com/googleapis/googleapis-discovery/commit/abf4cec1ce9e02e4d7d650bf66137c347cdd0d44
Source-Link: https://github.com/googleapis/googleapis-discovery/commit/6a189a8bada35e2657a67b903cad7fee649f2dbc

parthea commented:
Hi @m-strzelczyk, Please can you take a look?

m-strzelczyk commented:
https://github.com/googleapis/python-compute/pull/83 should take care of this - synthtool reset the config to use same project for tests, while those tests require separate projects. At first I made the mistake of changing that in `noxfile.py` as I didn't know the power of synthtool :P

yoshi-automation committed:
fix(monolith): Bump microgenerator versions to monolith removal

* fix(php): Bump microgenerator version

* fix(monolith): Bump microgenerator versions with monolith removal

Source-Author: Mira Leung <miraleung@users.noreply.github.com>
Source-Date: Wed Jun 23 12:35:59 2021 -0700
Source-Repo: googleapis/googleapis-discovery
Source-Sha: 6a189a8bada35e2657a67b903cad7fee649f2dbc
Source-Link: https://github.com/googleapis/googleapis-discovery/commit/6a189a8bada35e2657a67b903cad7fee649f2dbc

yoshi-automation committed:
chore!: Migrate to gapic-generator-java, remove monolith dependency

Source-Author: Vadym Matsishevskyi <25311427+vam-google@users.noreply.github.com>
Source-Date: Wed Jun 30 10:37:13 2021 -0700
Source-Repo: googleapis/googleapis-discovery
Source-Sha: abf4cec1ce9e02e4d7d650bf66137c347cdd0d44
Source-Link: https://github.com/googleapis/googleapis-discovery/commit/abf4cec1ce9e02e4d7d650bf66137c347cdd0d44

yoshi-automation committed:
fix!: regenerate competu.proto with uint64/int64 field types where appropriate (instead of string)

This is a breaking chagne for the clients

Source-Author: Vadym Matsishevskyi <25311427+vam-google@users.noreply.github.com>
Source-Date: Wed Jun 30 16:36:27 2021 -0700
Source-Repo: googleapis/googleapis-discovery
Source-Sha: c5bb8d324832f638d86bc1348fb45fb3d9c1fb30
Source-Link: https://github.com/googleapis/googleapis-discovery/commit/c5bb8d324832f638d86bc1348fb45fb3d9c1fb30

busunkim96 commented:
Closing to generate a new PR.
